examples/projections/table/README.txt

```diff
@@ -0,0 +1,84 @@
+Projection Table
+----------------
+
+PyGMT offers 31 map projections. These are specified using the ``projection`` argument.
+The projection codes for the projection styles are shown below. Upper case arguments
+set *map width* and lower case arguments set *map scale*.
```

Comment:
> Upper case arguments set *map width* and lower case arguments set *map scale*.

New users will be confused about what "Upper case" and "Lower case" mean. Please try to rephrase it, or just give an example.

Reply:
https://pygmt-git-fork-willschlitzer-proj-table.gmt.vercel.app/projections/index.html

I just realized that we already have similar sentences at the top of this page.

> PyGMT support many map projections. Use the projection argument to specify which one you want to use in all plotting modules. The projection is specified by a one letter code along with (sometimes optional) reference longitude and latitude and the width of the map (for example, Alon0/lat0[/horizon]/width). The map height is determined based on the region and projection.

Perhaps no need to repeat it in the `projections/table/README.txt` file

Reply:
I removed it from the table README